src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -48,6 +48,7 @@ public class Drivetrain extends SubsystemBase {
 
   private SlewRateLimiter m_magLimiter;
   private SlewRateLimiter m_rotLimiter;
+  private Vector spdCommanded;
```

Comment:
I think you need to instantiate this in the constructor.

Reply:
It's value is set before it's ever used, so it should be fine

Reply:
I agree with Emmy, this is bad practice.

Reply:
@Irishumanoid If you do this throughout the code you're going to have a lot of little asterisks that you're not going to remember. Better to always instantiate so you don't have to worry.

Reply:
fixed